File: CrimsonClubs/CrimsonClubs.Models/Dtos/GroupDto.cs
using CrimsonClubs.Models.Entities;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace CrimsonClubs.Models.Dtos
{
    public class GroupDto
    {
        public int Id { get; set; }
        public string Name { get; set; }
        public string Description { get; set; }
        public int OrganizationId { get; set; }
        public int ClubCount { get; set; }

        public GroupDto()
        {

        }

        public GroupDto(Group dbo)
        {
            Id = dbo.Id;
            Name = dbo.Name;
            Description = dbo.Description;
            OrganizationId = dbo.OrganizationId;
            ClubCount = dbo.Clubs.Count;
        }
    }
}

File: Android/CrimsonClubs/app/src/main/java/crimsonclubs/uacs/android/crimsonclubs/GroupDto.java
package crimsonclubs.uacs.android.crimsonclubs;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class GroupDto {

    @Expose
    @SerializedName("id")
    public int id;
    @Expose
    @SerializedName("name")
    public String name;
    @Expose
    @SerializedName("description")
    public String description;
    @Expose
    @SerializedName("OrganizationId")
    public int orgId;
    @Expose
    @SerializedName("clubcount")
    public int clubCount;

}
